testing_and_setup/compass/ocean/jigsaw_to_MPAS/3Wbgy5.xml

```diff
@@ -0,0 +1,55 @@
+<ColorMaps>
+<ColorMap space="CIELAB" indexedLookup="false" name="3Wbgy5">
+<Point x="0.000000" o="1" r="0.831373" g="0.909804" b="0.980392"/>
+<Point x="0.012500" o="1" r="0.749020" g="0.862745" b="0.960784"/>
+<Point x="0.025000" o="1" r="0.694118" g="0.827451" b="0.941176"/>
```

Comment:
It looks like you downloaded this directly from Paraview? That was smart.

Reply:
No, copied from MPAS-Analysis, which gives proper credit and got it from the original source (LANL's own SciVisColor web page, I believe).